Cargo.toml

```diff
@@ -57,6 +57,7 @@ once_cell = "1.13.0"
 
 [features]
 default = ["default-mechanisms", "clients-5"]
+ext = []
```

Comment:
This is too short and C (same actually for `virt`), perhaps `api-extensions`?

Reply:
NB: We chose `virt` because `virtual` is a keyword and `virtual_platform` seemed excessive, but if you have a better idea, I’ll change it.